src/platform/silabs/efr32/efr32-chip-mbedtls-config.h

```diff
@@ -81,10 +81,15 @@
 #define MBEDTLS_CIPHER_MODE_CBC
 #define MBEDTLS_CCM_C
 #define MBEDTLS_ECJPAKE_C
+#define MBEDTLS_X509_CRT_PARSE_C
 #define MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED
 #define MBEDTLS_PK_HAVE_CURVE_SECP256R1
 #define MBEDTLS_PK_HAVE_JPAKE
-#define PSA_WANT_ALG_SOME_PAKE
+#define MBEDTLS_ECP_DP_SECP256R1_ENABLED
+#define MBEDTLS_SSL_HANDSHAKE_WITH_CERT_ENABLED
+#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC
+#define PSA_WANT_ALG_JPAKE
+#define PSA_WANT_ECC_SECP_R1_256
```

Comment:
These should go in the PSA header instead of the mbedtls header.

Reply:
oh ok, yeah they can